setup.py

```diff
@@ -5,27 +5,21 @@
 import re
 
 
-v = open(os.path.join(os.path.dirname(__file__), 'ibm_db_sa', '__init__.py'))
+v = open(os.path.join(os.path.dirname(__file__), 'pyodbc_sa', '__init__.py'))
 VERSION = re.compile(r".*__version__ = '(.*?)'", re.S).match(v.read()).group(1)
 v.close()
 
-readme = os.path.join(os.path.dirname(__file__), 'README.rst')
-if 'USE_PYODBC' in os.environ and os.environ['USE_PYODBC'] == '1':
-    require = ['sqlalchemy>=0.7.3']
-else:
-    require = ['sqlalchemy>=0.7.3','ibm_db>=2.0.0']
+readme = os.path.join(os.path.dirname(__file__), 'README.md')
     
 
 setup(
-         name='ibm_db_sa',
+         name='pyodbc_sa',
          version=VERSION,
          license='Apache License 2.0',
-         description='SQLAlchemy support for IBM Data Servers',
+         description='SQLAlchemy support for IBM i Db2',
          author='IBM Application Development Team',
          author_email='opendev@us.ibm.com',
```

Comment:
This needs to be changed as we don't control this email.

Reply:
I think we can just remove it. Our python-itoolkit doesn't have it: https://github.com/IBM/python-itoolkit/blob/master/setup.py